verification/infrastructure/repositories/duns_repository.py

```diff
@@ -15,26 +15,28 @@ def add_verification(self, verification):
             status=verification.status, created_at=verification.created_at, updated_at=verification.updated_at)
         self.add_item(verification_db)
 
-    def _get_verification(self, verification_id=None, org_uuid=None):
+    def __get_verification(self, verification_id=None, org_uuid=None):
```

Comment:
Why two method with the same name? One is private and other is public.

Reply:
one is private which returns db entity of verification and one is public which returns domain entity. both function's purpose is the same, as in private it will return internal object and in public it will return client specific object